Cookbook/Technical-Documents/ReleaseProcess.md

```diff
@@ -49,13 +52,17 @@ There are usually two release engineers working at any given time. It goes witho
   1. On the sidebar click `+ Version or Platform` and select `iOS`.
   1. Input the new version number.
 
-### Phase 2: Test and fix bugs
+### Phase 3: Test and fix bugs
 *It starts after the App Center build has been delivered and it can take several cycles*
 
 1. Testers will then begin their work against the build you just created.
-1. Any hotfix should target that branch, and you, as the release engineer, are responsible for double checking that the hotfix's PR is pointing to the release branch (instead of `develop`). The issue for the hotfix should be added to the release JIRA board.
+1. Any hotfix should target the `develop` branch first. 
+1. These PRs need to be reviewed by the relevant squad or platform QA (Bibin Paul) & one of the release engineers assigned to the release. This is to ensure visibility of changes being requested to be made, and to ensure the correct builds are available for validation. The exception being that if an issue is not reproducible on `develop` then we can merge directly with `release`. Still, it should be fully understandable why it is not reproducible to help ensure there is no regression for the next release.
+    * Bear in mind that two approvals from other engineers assigned by Pull Assigners is not enough in this particular case.
+    * After merging, we'll need to cherry-pick the commit via a PR to the release branch. Doing this will ensure that both branches are up to date and reduces the risk of conflicts when merging the release branch back to develop. 
+    * Furthermore, the issue for the hotfix has to be added to the release JIRA board.
```

Comment:
```suggestion
    * Furthermore, the issue for the hotfix has to be added to the release JIRA board. This can be done automatically by setting the release number in `Fix version` in the hotfix's JIRA card.
```

Like this @AnnKatF ?

Reply:
Perfect!